### ISSUE TITLE
Aiming down sight increases accuracy, but can't be done while reloading shotgun, unlike aos 0.75

### ISSUE THREAD

**Issue by pielover88888** [yvt/openspades#729]
Pressing R while ADS'd in aos 0.75 doesn't un-ADS, but in openspades it does.

This is not great, giving shotgun players who constantly mash reload an accuracy disadvantage.

**Comment by NotAFile** [comment]
tbh, I don't think you should be able to reload while in ADS

Plus, OS shotgun users have an accuracy advantage already anyway (and that's good because .75 shotgun is useless)

**Comment by pielover88888** [comment]
In AoS 0.75, you can just spam R with shotgun, while ADS (increasing accuracy by 50%), and never run out of shots with no penalty. If AoSB compatibility is of priority, it should be the same in openspades. It would however be annoying to have to add a reloading animation that looks pretty while ADS.

**Comment by NotAFile** [comment]
I don't think just identically cloning AoSB in every aspect is a good goal. If we want to, finally, after 6 years get this game forward we must be ready to make changes that improve the game even if it's different from what the original client does. I think it makes more sense to look at the game the way it currently is and make decisions because they are good decisions, not just "because Ben did it that way".

**Comment by pielover88888** [comment]
> If we want to, finally, after 6 years get this game forward we must be ready to make changes that improve the game even if it's different from what the original client does. 

I agree, but only on the next generation of servers that will eventually not have AoS classic support. Until then, we should be on equal footing in terms of gunplay. It's not as if shotgun needs nerfs.

**Comment by BozoDel** [comment]
> I don't think just identically cloning AoSB in every aspect is a good goal. If we want to, finally, after 6 years get this game forward we must be ready to make changes that improve the game even if it's different from what the original client does. I think it makes more sense to look at the game and how the change fits into it and make decisions because they are good decisions, not just "because Ben did it that way".
Eh, I disagree. I'm all for an updated standard (let's call it 0.77), but if the client claims to be compatible with 0.75/0.76, I'd expect it to be.
